css-layout-classes/styles.css

```diff
@@ -0,0 +1,28 @@
+* {
+  box-sizing: border-box;
+}
+
+img {
+  width: 100%;
+}
+
+.container {
+  width: 90%;
```

Comment:
as per the hints in the instructions, this should be set to a max-width of 800px and you should have margin set to 0 auto to center the container on the page